ash-window/src/lib.rs

```diff
@@ -171,3 +171,69 @@ pub fn enumerate_required_extensions(
 
     Ok(extensions)
 }
+
+/// Query whether a `queue_family` of the given `physical_device` supports presenting to any surface that might be created.
+/// This function can be used to find a suitable [`vk::PhysicalDevice`] and queue family
+/// for rendering before a single surface is created.
+///
+/// This function can be a more useful alternative for
+/// [`VkGetPhysicalDeviceSurfaceSupportKHR`](https://registry.khronos.org/vulkan/specs/1.3-extensions/html/chap34.html#vkGetPhysicalDeviceSurfaceSupportKHR),
```

Comment:
1. This is a function, and starts with a _lowercase_ `v`;
2. It is preferred to have (long) links at the bottom, so that the source text isn't too crowded and oddly flowed (``[`vkGetPhysicalDeviceSurfaceSupportKHR`]``) automatically crates a label that you can assign a link to via ``/// [`vkGetPhysicalDeviceSurfaceSupportKHR`]: https://...``
   (More important below);
3. Since Ash is a dependency (and in scope via `ash::extensions::khr`), prefer linking to the Rust implementation of this function via ``[`khr::Surface::get_physical_device_surface_support()`]``.

(And separately link the docs page _if_ there's more useful information to describe explicitly)